code/modules/surgery/organs/external/wings/functional_wings.dm

```diff
@@ -199,3 +199,8 @@
 	name = "fly wings"
 	desc = "Fly as a fly."
 	sprite_accessory_override = /datum/sprite_accessory/wings/fly
+
+/obj/item/organ/external/wings/functional/clockwork
```

Comment:
same with this

Reply:
i don't know what you mean by modular folder, so i moved them back into disease file until future notice.